src/web/routes/application/send-code/send-code.js

```diff
@@ -10,11 +11,17 @@ const pageContent = ({ translate }) => ({
   explanation: translate('sendCode.explanation')
 })
 
+const behaviourForPost = (req, res, next) => {
+  req.session[CONFIRMATION_CODE_SESSION_PROPERTY] = '123456'
```

Comment:
I've not written any tests for this as it will change soon to use a unique code, not a constant